test/index.spec.ts

```diff
@@ -78,16 +78,11 @@ describe("addForwarder", () => {
       handler: "hello.handler",
     });
     addForwarder(stack, [nodeLambda, pythonLambda], "forwarder-arn");
```

Comment:
I noticed that the `Subscribes the same forwarder to two different lambda functions via separate addLambdaFunctions function calls` test uses `datadogCdk.addLambdaFunctions([nodeLambda]);` and `datadogCdk.addLambdaFunctions([pythonLambda]);` to add the subscriptions however this test directly uses `addForwarder`. Could you update the tests to use the `addForwarder` function instead of the `datadogCdk.addLambdaFunctions()` way? 

This way the convention in the file is the same and so the `addForwarder` set of tests really only test the `addForwarder` function and not all the other codepaths that come with using `datadogCdk.addLambdaFunctions()`

Reply:
I didn't want to get too deep into reworking the tests, but seems best to move all addForwarder specific tests to the forwarder.spec.ts, and factor out the "search the cdk nodes" logic to be reused by both index.spec and forwarder.spec

Let me know how you feel about the most recent commit!